src/__experimental__/components/fieldset/fieldset-classic.style.js

```diff
@@ -0,0 +1,11 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+
+export default ({ theme }) => theme.name === THEMES.classic && css`
+    color: #003349;
+    font-size: 14px;
```

Comment:
is it possible to provide this value like theme.text.size?

Reply:
Definitely possible, there seems to be a movement away from updating the `classic` theme config though, unless I'm wrong?